Makefile

```diff
@@ -222,6 +226,12 @@ cluster/prepare/dms:
 	@-oc create secret generic $(INSTALLATION_PREFIX)-deadmanssnitch -n $(NAMESPACE) \
 		--from-literal=url=https://dms.example.com
 
+.PHONY: cluster/prepare/delorean
+cluster/prepare/delorean:
+ifneq ( ,$(findstring image_mirror_mapping,$(IMAGE_MAPPINGS)))
```

Comment:
@laurafitzgerald where is the `IMAGE_MAPPINGS` variable coming from now?

Reply:
@b1zzu it comes from here: https://github.com/integr8ly/integreatly-operator/blob/master/make/manifest.mk#L1

Reply:
Does this mean that when make run all .mk files are also sourced? Because I couldn't find anyway the two files are linked together

Reply:
https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L1